### ISSUE TITLE
Host mandatory in swagger_helper.rb

### ISSUE THREAD

**Issue by HDLP9** [rswag/rswag#587]
Hi,

We just lost a couple of hours with this issue and I decided to open an issue just to check if its something that we're doing wrong or there's something missing in the documentation about this.

**Steps to reproduce:**

- Create a new rails api project
- Add rswag, bundle and install it
- Generate any controller or scaffold like Article
- Create an integration spec to describe and test that controller
- in swagger_helper.rb add a server with url and default host (like api.foo.com)
- run SWAGGER_DRY_RUN=0 rails rswag:specs:swaggerize and watch every request going to example.com

`...,
servers: [
{
    url: 'http://{defaultHost}',
    variables: {
      defaultHost: {
        default: 'api.foo.com'
      }
    }
  },
]`

Tried adding it with c.swagger_filter without any success

**Workaround:**

- in swagger_helper.rb add a key "host" 
- run SWAGGER_DRY_RUN=0 rails rswag:specs:swaggerize and watch every request going to the right domain

`...,
host: 'api.foo.com',
servers: [
  {
    url: 'http://{defaultHost}',
    variables: {
      defaultHost: {
        default: 'api.foo.com'
      }
    }
  },
]`

**Comment by mirkode** [comment]
I have the same problems, however adding the `host` key is not working.
Whatever I do, it is defaulting to `www.example.com:80` and it is driving me slightly nuts :-)

Not able to run the tests without failing. Maybe I am just doing something wrong, but I cannot see where.
I stumbled upon the same issue this [PR](https://github.com/rswag/rswag/pull/591) is fixing, but once I patched that it was still failing with this sticky `example.com` host.

Any help is greatly appreciated!
